doc/nrf/libraries/bluetooth_services/services/fast_pair.rst

```diff
@@ -137,16 +130,12 @@ The Fast Pair specification requires support for Bluetooth® Low Energy pairing
 Firmware Revision characteristic
 --------------------------------
 
-The Fast Pair specification requires enabling GATT Device Information Service and the Firmware Revision characteristic.
-For this reason, the default values of the Kconfig options :kconfig:option:`CONFIG_BT_DIS` and :kconfig:option:`CONFIG_BT_DIS_FW_REV`, respectively, are set to enabled.
-The default value of :kconfig:option:`CONFIG_BT_DIS_FW_REV_STR` is set to :kconfig:option:`CONFIG_MCUBOOT_IMGTOOL_SIGN_VERSION` if :kconfig:option:`CONFIG_BOOTLOADER_MCUBOOT` is enabled.
-The option is enforced by sysbuild when ``SB_CONFIG_BOOTLOADER_MCUBOOT`` is enabled.
+The Fast Pair specification requires enabling GATT Device Information Service and the Firmware Revision characteristic for selected Fast Pair use cases (for example, the input device use case).
 
+For this reason, the relevant use case Kconfig options (for example, the :kconfig:option:`CONFIG_BT_FAST_PAIR_USE_CASE_INPUT_DEVICE` Kconfig option) select the :kconfig:option:`CONFIG_BT_DIS` and :kconfig:option:`CONFIG_BT_DIS_FW_REV` Kconfig options.
+If the target project uses Zephyr's :ref:`zephyr:app-version-details`, the default value of the :kconfig:option:`CONFIG_BT_DIS_FW_REV_STR` Kconfig option is set according to the versioning information found in the :file:`VERSION` file.
```

Comment:
Maybe we could link here: https://docs.zephyrproject.org/latest/build/version/index.html instead of just mentioning `VERSION` file?